egs2/fisher_callhome_spanish/asr1/local/data.sh

```diff
@@ -69,7 +64,17 @@ if [ ${stage} -le 1 ] && [ ${stop_stage} -ge 1 ]; then
 
     # concatenate multiple utterances
     local/normalize_trans.sh ${sfisher_transcripts} ${callhome_transcripts}
-    ./utils/combine_data.sh data/dev_all data/dev data/dev2
+
+    utils/combine_data.sh \
+        --extra-files "text.lc.en text.lc.es text.lc.rm.en text.lc.rm.es text.tc.en text.tc.es" \
+        data/train \
+        data/fisher_train data/callhome_train/ 
+
+    cp -r data/fisher_dev data/dev
+    # Use 1 reference as dev set
+    ln -sf data/dev/text.lc.en.0 data/dev/text.lc.en
```

Comment:
The same as last comments, just use `text` would be fine (having others won't function for `asr.sh`

Reply:
Removed. Thanks!